extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/AuthRuntimeConfig.java

```diff
@@ -76,4 +76,48 @@ public interface AuthRuntimeConfig {
      * Form Auth config
      */
     FormAuthRuntimeConfig form();
+
+    /**
+     * Require that all registered HTTP authentication mechanisms must complete the authentication.
+     * <p>
+     * Typically, this property has to be true when the credentials are carried over mTLS, when both mTLS and another
```

Comment:
I think we now have remove some of this content, related to MTLS, OIDC. Instead, let's say something like `By default, when the `{@link #inclusiveMode}` is  `strict`, every registered authentication mechanism must produce `SecurityIdentity`, otherwise, a number of mechanisms which produced the identity may be less than a total number of registered mechanisms`.

Then explain how to access `SecurityIdentity` when more than identity is available, and mention that for example, when MTLS is used, it is always going to be the first one...

Someting like that, does it sound reasonable ?

Reply:
Well, I really like the original text, so I moved it to the strict mode constant with little adjustment. Otherwise it does.